controllers/apps/apimanager_controller.go

```diff
@@ -48,22 +48,22 @@ type APIManagerReconciler struct {
 // blank assignment to verify that APIManagerReconciler implements reconcile.Reconciler
 var _ reconcile.Reconciler = &APIManagerReconciler{}
 
-// +kubebuilder:rbac:groups=apps.3scale.net,namespace=placeholder,resources=apimanagers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Same should be applied to other controllers? 3scale operator has a number of those

Reply:
We reverted these rule changes,  as OLM handles RBAC